packages/core/src/helpers/auth.ts

```diff
@@ -42,7 +46,7 @@ export enum FederatedIdentityProviders {
  */
 export const getAliasInfoFromContext = (context: AuthContext) => {
   const loginMechanisms = context.config?.login_mechanisms ?? ['username'];
-  const error = context.validationError['username'];
+  const error = context.actorRef?.context?.validationError['username'];
```

Comment:
Question here: I think it could be useful to have a constant referencing the string `'username'`, but it's a little confusing since `validationError['username']` isn't really related to how we use `'username'` as the default `login_mechanisms`. Any thoughts on how we can make our use of `'username'` a bit clearer?

Reply:
Ah, good catch here. And I believe this implementation is wrong -- "email" / "password" are not stored as "email" / "password" when on `signUp` screen. We just never had problems with this because `username` was never verified. 

Let's keep this as now, but let's consider making every username alias stored as "username", and we can disambiguate them elsewhere so we have consistent `formValues` interface.